src/uct/cuda/cuda_ipc/cuda_ipc.cuh

```diff
@@ -107,6 +93,19 @@ UCS_F_DEVICE void uct_cuda_ipc_level_sync()
     return;
 }
 
+UCS_F_DEVICE void* uct_cuda_ipc_map_remote(const uct_cuda_ipc_device_mem_element_t* elem,
+                                           uint64_t remote_address)
+{
+    return reinterpret_cast<void*>((uintptr_t)remote_address + elem->mapped_offset);
+}
+
+UCS_F_DEVICE void uct_cuda_ipc_atomic_inc(uint64_t *dst, uint64_t inc_value)
+{
+    cuda::atomic_ref<uint64_t, cuda::thread_scope_system> dst_ref{*dst};
+    dst_ref.fetch_add(inc_value, cuda::memory_order_relaxed);
+    cuda::atomic_thread_fence(cuda::memory_order_release, cuda::thread_scope_system);
+}
+
```

Comment:
why need to move?

Reply:
uct_cuda_ipc_get_lane and uct_cuda_ipc_level_sync look nice together